pandas/core/strings.py

```diff
@@ -1095,24 +1095,48 @@ def str_pad(arr, width, side='left', fillchar=' '):
 
 def str_split(arr, pat=None, n=None):
     """
+    Split strings around given separator/delimiter.
+
     Split each string (a la re.split) in the Series/Index by given
     pattern, propagating NA values. Equivalent to :meth:`str.split`.
 
     Parameters
```

Comment:
General comment - rather than putting sub-bullets under the parameters would it be clearer to move those into a dedicated Notes section? They do explain some of the implementation details so wonder if they are better served there

Reply:
That would be better. I'll update it.